ideascube/models.py

```diff
@@ -55,6 +55,19 @@ def create_superuser(self, serial, password, **extra):
         user.save(using=self._db)
         return user
 
+    def get_queryset(self):
+        return super().get_queryset().exclude(serial='__system__')
+
+    def get_queryset_unfiltered(self):
+        return super().get_queryset()
+
+    def all(self, really=False):
```

Comment:
Isn't that exactly what kwargs are for, optionally influencing the behaviour of a method?

I don't mind adding a second method, bu it just feels like a waste of a perfectly good and pythonic coding pattern.